lib/msal-browser/test/app/PublicClientApplication.spec.ts

```diff
@@ -581,6 +577,29 @@ describe("PublicClientApplication.ts Class Unit Tests", () => {
 
             expect(preGenerateSpy).toHaveBeenCalledTimes(1);
         });
+
+        it("clears previous cache with claims", async () => {
```

Comment:
@sameerag Added this test to make sure cache with claims is cleared. This previously would not have been done if `claimsBasedCachingEnabled` was set to true.